inc/cropper/namespace.php

```diff
@@ -180,6 +188,13 @@ function attachment_js( $response, $attachment ) {
 	$meta         = wp_get_attachment_metadata( $attachment->ID );
 	$backup_sizes = get_post_meta( $attachment->ID, '_wp_attachment_backup_sizes', true );
 
+	// Ensure width and height are set.
```

Comment:
IMO we should not do this, and we want a fail state. We don't want to be calling `getimagesize` as that causes a remote fetch from s3 for each image. If the image width / height isn't set, something is wrong, and I think it's ok to fail in those cases.

Reply:
Yeah I was reading up on `getimagesize` performance after the fact. It's a rare occurrence but also it's not actually related to the problem at hand. Will remove.

Reply:
@joehoyle changed my mind - updated to return early but logging an error that tells you what image ID has an issue.